packages/ragbits-conversations/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## Unreleased
+- correct the typo in the class name ObjectConstructionConfig
```

Comment:
You should now be able to replace this with `Changelog-ignore: ragbits-conversations` in the commit message